### PR TITLE
Standalone lib

### PR DESCRIPTION
**Pull request by joe-s-avaya**
We (Enrique and Joe) have replaced the existing content with what we believe is the appropriate skeleton. We have implemented a bare-bones Angular library, some base workflows (based off of acilie9's work), and a fair amount of documentation on the root readme.

Important Notes:
- there is ONLY a library here
- I simplified the two workflows that acilie9 created
- we added several documents in `./readmes/` that are mostly copy pasted from neo-react, please modify as you wish
- we added an issue template and a pull request template, these are also copied from neo-react. We have found them extremely helpful.
- we use strict versioning in the `package.json` as we've been bitten by having loose versioning.
- we're using straight `npm` for this project, feel free to update to use yarn/pnpm/bun/whichever
- I copy-pasted the license files that I got from legal. They are effectively the same as the ones that acilie9 added. 
- `neo-angular-library` was the project and librarie's name, we have thus renamed the library to `neo-standalone-components`. What is published is still [@avaya/neo-angular](https://www.npmjs.com/package/@avaya/neo-angular)

### CODE REVIEW

**Review by enrique-prado — APPROVED** [review]
Left a few comments and questions.

File: readmes/how-to-publish.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+Once that is complete, the next step is to add the release to our GitHub
+- navigate to our [GitHub releases page](https://github.com/avaya-dux/neo-angular-library/releases)
+- click: "Draft a new release" (top right corner)
+- choose tag (or create a new tag on publish), using format `vX.X.X`
+- target `main` branch
+- add title using format: "v0.1.0 (Jan 18th 2024)"
+- click: "Auto-generate release notes"
+- attach the binary generated from `yarn all` (tgz file)
```
[the remark this answers is not in the document]
Comment (joe-s-avaya) [reply readmes/how-to-publish.md]: updated

File: projects/neo-standalone-components/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
```
[the remark this answers is not in the document]
Comment (joe-s-avaya) [reply projects/neo-standalone-components/.eslintrc.json:1]: discussed IRL

File: projects/neo-standalone-components/LICENSE-3rd-party.html [hunk 3]
```diff
@@ -1,35 +1,16 @@
-<html>
+<html lang="en">
    <head></head>
    <body>
       <H1><font color = "black"> <u>Application License & Copyright Text Report</u></font></H1>
       <BR>
-      <H1><b><font color = "black"> Software Release: Neo Angular Component Library Versions: 17.0.0</font></b></H1>
-      <p><b>Release Date: 2023-11-06</b></p>
+      <H1><b><font color = "black"> Software Release: Neo Angular Component Library Versions: 1.1.3</font></b></H1>
```
[the remark this answers is not in the document]
Comment (joe-s-avaya) [reply projects/neo-standalone-components/LICENSE-3rd-party.html]: good call, removing version

**Pull request merged**